packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/useLocationDetailView.ts

```diff
@@ -21,19 +21,27 @@ interface UseLocationDetailView {
   isLoading: boolean;
   isPaginateNextDisabled: boolean;
   isPaginatePreviousDisabled: boolean;
+  currentLocation: LocationData | undefined;
```

Comment:
when will currentLocation be `undefined`?

Reply:
Theoretically, on the views it is used, never. But it is technically possible for it to be undefined on the location state. Rather than just doing some one size fits all cut off in the hook, keeping this allows the downstream consumers to decide how best to deal it.